src/encoded/schemas/raw_matrix_file.json

```diff
@@ -0,0 +1,384 @@
+            "title": "Intronic reads counted",
+            "description": "True if reads mapped to intronic regions were used for gene quantifications.",
+            "type": "boolean"
+        },
+        "feature_keys": {
+            "title": "Feature keys",
+            "description": "The category of identifiers used as the key for the features of the matrix.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Feature key",
+                "type": "string",
+                "enum": [
+                    "Ensembl gene ID",
+                    "gene symbol",
+                    "antibody target",
+                    "genomic coordinates",
+                    "motif ID"
+                ]
+            }
+        },
+        "feature_counts": {
+            "title": "Feature counts",
+            "description": "For each category of feature type, the number of entries found in the matrix considered as features (eg: genes, antibody capture, or peaks)",
+            "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Feature count",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "feature_type": {
+                        "title": "Feature type",
+                        "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+                        "type": "string",
+                        "enum": [
+                            "gene",
+                            "peak",
+                            "transcription factor",
+                            "antibody capture"
+                        ]
+                    },
+                    "feature_count": {
+                        "title": "Feature count",
+                        "comment":  "Do not submit. This will be filled in during checkfiles validation.",
+                        "description": "",
+                        "type": "integer"
+                    }
+                }
+            }
+        },
+        "value_scale": {
+            "title": "Value scale",
+            "description": "The factor by which the expression values have been scaled; linear if not scaled.",
+            "type": "string",
+            "default": "linear"
+        },
+        "value_units": {
+            "title": "Value units",
+            "description": "The units for the expression values.",
+            "type": "string",
+            "enum": [
+                "UQ",
+                "SF",
+                "RPM",
+                "RPKM",
+                "FPKM",
+                "TPM",
+                "TMM",
+                "UMI",
+                "expected count",
+                "fragment ends"
+            ]
+        },
+        "normalized": {
```

Comment:
Not sure where this goes, but value_scale should equal linear, and normalized and scaled should be false for raw.

Reply:
I had them each as 'default', but better to put them as calculated fields so won't even be submittable. Updated, you'll see them in file.py now